rest/taskana-rest-spring/src/main/java/pro/taskana/rest/TaskCommentController.java

```diff
@@ -153,34 +146,22 @@
   @PostMapping(path = Mapping.URL_TASK_COMMENTS)
```

Comment:
Should this path for create remain as /tasks/comments/{taskId}/{taskCommentId} or also be changed to /tasks/comments/{taskCommentId}? @holgerhagen

Reply:
I was expecting  POST "/tasks/{taskId}/comments"